rtb/verbali/esterni/VE_2024-12-13_ver1.0.0.typ

```diff
@@ -0,0 +1,75 @@
+#import "/templates/template.typ": *
+
+#show: content=>verbale(
+  data: "13 dicembre 2024", 
+  responsabile: "Davide Martinelli", 
+  destinatari: (
+    "Prof. Tullio Vardanega",
+    "Prof. Riccardo Cardin",
+    "Sync Lab S.r.l."
+  ),
+  uso: "Esterno",
+  redattori: ("Davide Picello",), 
+  titolo: "SAL fine secondo sprint", 
+  verificatori: (
+    "Klaudio Merja",
+  ),
+  riassunto: [Incontro relativo alla fine del secondo _sprint_ che ha visto il gruppo presentare una prima implementazione concreta del simulatore dati con interfaccia],
+  versioni: (
+    "1.0.0", 
+    [16/12/2024], 
+    "Davide Picello", 
+    "Klaudio Merja",
+    [- Stesura del verbale],
+  ),
+  content: content,
+  firmaRichiesta: true,
+)
+
+
+#infoRiunione(luogo: "Google Meet", data: "13/12/2024", ora: "10:30", durata: "30 minuti", partecipantiEsterni: ("Andrea Dorigo",))
+
+
+
+= Sintesi dell'incontro 
+L'incontro ha ufficialmente segnato la fine del secondo _#rifGlossario("sprint")_, ed ha visto il gruppo presentare per la prima volta una proposta concreta del simulatore dati con interfaccia, utilizzando le tecnologie concordate.
+
+In particolare è stata mostrata la progettazione del _#rifGlossario("database")_, presentando il modello entità-relazione, a cui è seguito un breve confronto tra il gruppo ed il partecipante esterno, con esito positivo. 
+
+//Mostrato Nifi + Grafana
+Infine è stato esposto il generatore dati ed il suo funzionamento mostrando i flussi di dati in #rifGlossario("Apache NiFi"), e successivamente l'interfaccia realizzata con #rifGlossario("Grafana"). Quest'ultima presentava una mappa in cui venivano indicati i punti geospaziali creati dal generatore. Queste coordinate simulavano un percorso di un utente in bicicletta in modo corretto e coerente, come possiamo notare nelle seguenti immagini.
```

Comment:
Dopo ..flussi di dati in Apache Nifi... toglierei come sempre anche qui la virgola